x/auth/tx/textual.go

```diff
@@ -0,0 +1,70 @@
+package tx
+
+import (
+	"fmt"
+
+	"cosmossdk.io/tx/textual/valuerenderer"
+	signingtypes "github.com/cosmos/cosmos-sdk/types/tx/signing"
+	"google.golang.org/protobuf/types/known/anypb"
+
+	textualv1 "cosmossdk.io/api/cosmos/msg/textual/v1"
+	signingv1beta1 "cosmossdk.io/api/cosmos/tx/signing/v1beta1"
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/signing"
+)
+
+// signModeTextualHandler defines the SIGN_MODE_TEXTUAL SignModeHandler
+type signModeTextualHandler struct {
+	t valuerenderer.Textual
```

Comment:
I'm thinking of moving everything from `tx/textual/valuerender` to `tx/textual`, this way we have:
- `textual.Textual`
- `textual.ValueRenderer`
- `textual.NewAnyValueRenderer`

etc

Reply:
SGTM.